crates/bevy_window/src/event.rs

```diff
@@ -2,10 +2,19 @@ use std::path::PathBuf;
 
 use super::{WindowDescriptor, WindowId};
 use bevy_math::{IVec2, Vec2};
+use bevy_reflect::{FromReflect, Reflect};
+
+#[cfg(feature = "serialize")]
+use bevy_reflect::{ReflectDeserialize, ReflectSerialize};
 
 /// A window event that is sent whenever a window's logical size has changed.
-#[derive(Debug, Clone, PartialEq)]
-#[cfg_attr(feature = "serialize", derive(serde::Serialize, serde::Deserialize))]
+#[derive(Debug, Clone, PartialEq, Reflect, FromReflect)]
+#[reflect(Debug, PartialEq)]
+#[cfg_attr(
+    feature = "serialize",
+    derive(serde::Serialize, serde::Deserialize),
+    reflect(Serialize, Deserialize)
+)]
```

Comment:
unless you feel heavily towards this style, I would prefer to keep it consistent across Bevy on a single line

Reply:
oh never mind, this also adds the reflect attribute... Feels too heavy to have it like that though...

Reply:
Would you prefer splitting it up into two lines? I'm fine with either, but went with this approach to remove the redundant `feature = "serialize"`.

Reply:
I'm going to leave this as is for now, but I'm not strongly in favor of either syntax. I like that this syntax groups together the related attributes and reduces duplication, but I also agree that the syntax is pretty bulky. One way of viewing it, though, is that the attributes in the `cfg_attr` are "within the scope of the `serialize` feature".